pwiz_tools/Skyline/Controls/Graphs/GraphSpectrum.cs

```diff
@@ -1324,23 +1377,25 @@ private void DoUpdate()
                         else
                         {
                             mirrorSpectrum = null;
+                            MirrorGraphItem = null;
                         }
 
                         spectrumChanged |= !Equals(_mirrorSpectrum?.SpectrumInfo, mirrorSpectrum?.SpectrumInfo);
                         _mirrorSpectrum = mirrorSpectrum;
 
                         double? dotp = null;
+                        // need to keep it at the class level to be able to extract tooltip info.
                         double? fullDotp = null;
                         SpectrumGraphItem mirrorGraphItem = null;
                         if (mirrorSpectrum != null)
                         {
                             var peaksInfo = spectrum != null
                                 ? RescaleMirrorSpectrum(mirrorSpectrum, spectrum)
                                 : mirrorSpectrum.SpectrumPeaksInfo;
-                            mirrorGraphItem = MakeGraphItem(mirrorSpectrum, selection, settings, peaksInfo);
-                            mirrorGraphItem.Invert = true;
+                            MirrorGraphItem = MakeGraphItem(mirrorSpectrum, selection, settings, peaksInfo);
```

Comment:
The local variable `mirrorGraphItem` is still being used in a few places in this function and causes NullReferenceException's.
Those need to be changed to the capital `MirrorGraphItem`